pybossa/cache/projects.py

```diff
@@ -146,7 +146,8 @@ def format_task(row, lock_users=[]):
                 task.worker_filter,
                 task.worker_pref
                 FROM task
-                WHERE task.project_id =:project_id""" + filters
+                WHERE task.project_id =:project_id""" + filters +\
+                " ORDER BY %s" % (args.get('order_by') or 'priority_0 desc')
```

Comment:
Does args['order_by'] when present also has sorting order that toggles between ASC, DESC upon column click? If not, does code not need to handle sorting order that will toggle upon column clicks?

Reply:
> Does args['order_by'] when present also has sorting order that toggles between ASC, DESC upon column click?

Yes, user may pass `order_by` to sort the columns they want.